addon/authenticators/token.js

```diff
@@ -146,10 +146,16 @@ export default Base.extend({
     @return {object} An object with properties for authentication.
   */
   getAuthenticateData: function(credentials) {
-    var authentication = {};
-    authentication[this.passwordField] = credentials.password;
-    authentication[this.identificationField] = credentials.identification;
-    return authentication;
+    var password = credentials.password;
```

Comment:
@RuslanZavacky Any specific reason for rewriting this block? Not a big fan of modifying the `credentials` object that's passed in by doing `delete ...`.